inference-engine/src/mkldnn_plugin/nodes/mkldnn_math_node.cpp

```diff
@@ -0,0 +1,271 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cmath>
+#include <vector>
+#include <string>
+#include <mkldnn_types.h>
+#include "ie_parallel.hpp"
+#include <mkldnn_selective_build.h>
+#include "mkldnn_math_node.h"
+#include <nodes/common/tensor_desc_creator.h>
+#include <ngraph/opsets/opset2.hpp>
+#include "utils/general_utils.h"
```

Comment:
Please remove unused includes

Reply:
Fixed in 8c979d2088b5538bf6caf32c81e30a5b9dd485d1. Thank you!